internal/common/env.go

```diff
@@ -0,0 +1,9 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
In most cases, directories like `common`, `utils`, `tools` are considered to be an antipattern. Do you think that you can place it in a more specific package?

Reply:
Totally agreed, I'll check to move it to another package (or create a new one)